src/test/java/org/ibp/api/brapi/v2/program/ProgramResourceBrapiv2Test.java

```diff
@@ -203,20 +203,6 @@ public void testListProgramFilterByName() throws Exception {
 		;
 	}
 
-	@Test
```

Comment:
@cuenyad 
Not clear why this test is deleted if it is about BrapI?

Reply:
Reverted change and fixied test.